src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package mate.academy.bookstore.dto;
 
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull(message = "can't be null")
     private String title;
+    @NotNull(message = "can't be null")
     private String author;
+    @NotNull(message = "can't be null")
     private String isbn;
```

Comment:
ISBN field also has some other validation rules (like a format and size). So, validate it too.